test/runtests.jl

```diff
@@ -30,6 +30,23 @@ end
 @test !isnothing(1)
 @test isnothing(nothing)
 
+# https://github.com/JuliaLang/julia/pull/29749
+if VERSION > v"0.7"
```

Comment:
```suggestion
if VERSION >= v"0.7"
```
or the tests won't be executed on 0.7.